packages/core/src/App/Components/Elements/LandscapeBlocker/landscape-blocker.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { isDisabledLandscapeRoute, isTabletOs, routes } from '@deriv/shared';
+import './landscape-blocker.scss';
+
+const LandscapeBlocker = () => {
+    const location = useLocation();
+    const pathname = location?.pathname;
+    const is_hidden_landscape_blocker = isDisabledLandscapeRoute(pathname);
+    const shouldShowDtraderTabletView = pathname === routes.trade && isTabletOs;
+
+    if (is_hidden_landscape_blocker || shouldShowDtraderTabletView) return null;
+
+    return (
+        <div id='landscape_blocker' className='landscape-blocker'>
+            <div className='landscape-blocker__icon'>
+                <svg xmlns='http://www.w3.org/2000/svg' width='80' height='62' fill='none'>
```

Comment:
can we make this svg as a separate file to keep the code clean?

Reply:
@farhan-nurzi-deriv already did this in my [PR](https://github.com/binary-com/deriv-app/pull/13396)